examples/with-google-analytics/pages/_app.js

```diff
@@ -14,7 +15,30 @@ const App = ({ Component, pageProps }) => {
     }
   }, [router.events])
 
-  return <Component {...pageProps} />
+  return (
+    <>
+      {/* Global Site Tag (gtag.js) - Google Analytics */}
+      <Script
+        strategy="afterInteractive"
+        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_TRACKING_ID}`}
+      />
+      <Script
+        strategy="afterInteractive"
+        dangerouslySetInnerHTML={{
+          __html: `
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+
+            gtag('config', '${gtag.GA_TRACKING_ID}', {
+              page_path: window.location.pathname,
+            });
+          `,
+        }}
+      />
+      <Component {...pageProps} />
+    </>
+  )
```

Comment:
@janicklas-ralph @rgabs Our ESLint rule [doc](https://github.com/vercel/next.js/blob/canary/errors/next-script-for-ga.md#using-gtagjs) suggests to use `lazyOnLoad` when loading analytics via `gtag`. We need to make sure our messaging is aligned with the approach used in all the `examples/`.

Reply:
Updated @housseindjirdeh 👍

Reply:
afterInteractive (default): For scripts that can fetch and execute after the page is interactive, such as tag managers and analytics. These scripts are injected on the client-side and will run after hydration.

[Docs page](https://nextjs.org/docs/basic-features/script)
This make me confuse!
Should I use default strategy(afterInteractive) or lazyOnLoad?

Reply:
Updated that doc as well on this PR

Reply:
Thanks @smakosh 👏

Reply:
Thanks @smakosh, `afterInteractive` is the correct approach so appreciate you updating the documentation to make them aligned :)